test/perf/imgpkg.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package perf
```

Comment:
seems like we are using same type of functionality. can we find a different method to section off the tests? (if that's what the intent was to do with a separate package).

Reply:
Spoke to @cppforlife and sharing the imgpkg run and logger into a shared package between e2e and perf

Reply:
I decided to move every utility file (even those that are not shared across the 2 packages) into a 'helpers' package.

It results in only test files living under e2e / perf, which I think is cleaner because it helps to see what is being tested at a glance.